modules/seqexec/server/src/main/scala/seqexec/server/keywords/package.scala

```diff
@@ -3,84 +3,157 @@
+  object TypeInt16   extends KeywordType("INT16", "INT")
+  object TypeInt32   extends KeywordType("INT32", "INT")
+  object TypeFloat   extends KeywordType("FLOAT", "DOUBLE")
+  object TypeDouble  extends KeywordType("DOUBLE", "DOUBLE")
+  object TypeBoolean extends KeywordType("BOOLEAN", "BOOLEAN")
+  object TypeString  extends KeywordType("STRING", "STRING")
 
   // The developer uses these classes to define all the typed keywords
-  sealed class Keyword[T] protected (val n: String, val t: KeywordType, val v: T)
-  final case class Int8Keyword(name: String, value: Byte) extends Keyword[Byte](name, TypeInt8, value)
-  final case class Int16Keyword(name: String, value: Short) extends Keyword[Short](name, TypeInt16, value)
-  final case class Int32Keyword(name: String, value: Int) extends Keyword[Int](name, TypeInt32, value)
-  final case class FloatKeyword(name: String, value: Float) extends Keyword[Float](name, TypeFloat, value)
-  final case class DoubleKeyword(name: String, value: Double) extends Keyword[Double](name, TypeDouble, value)
-  final case class BooleanKeyword(name: String, value: Boolean) extends Keyword[Boolean](name, TypeBoolean, value)
-  final case class StringKeyword(name: String, value: String) extends Keyword[String](name, TypeString, value)
+  sealed class Keyword[T] protected (val n: String,
+                                     val t: KeywordType,
+                                     val v: T)
+  final case class Int8Keyword(name: String, value: Byte)
+      extends Keyword[Byte](name, TypeInt8, value)
+  final case class Int16Keyword(name: String, value: Short)
+      extends Keyword[Short](name, TypeInt16, value)
+  final case class Int32Keyword(name: String, value: Int)
+      extends Keyword[Int](name, TypeInt32, value)
+  final case class FloatKeyword(name: String, value: Float)
+      extends Keyword[Float](name, TypeFloat, value)
+  final case class DoubleKeyword(name: String, value: Double)
+      extends Keyword[Double](name, TypeDouble, value)
+  final case class BooleanKeyword(name: String, value: Boolean)
+      extends Keyword[Boolean](name, TypeBoolean, value)
+  final case class StringKeyword(name: String, value: String)
+      extends Keyword[String](name, TypeString, value)
 
   // At the end, I want to just pass a list of keywords to be sent to the DHS. I cannot do this with Keyword[T],
   // because I cannot mix different types in a List. But at the end I only care about the value as a String, so I
   // use an internal representation, and offer a class to the developer (KeywordBag) to create the list from typed
   // keywords.
 
-  private[keywords] final case class InternalKeyword(name: String, keywordType: KeywordType, value: String)
+  private[keywords] final case class InternalKeyword(name: String,
+                                                     keywordType: KeywordType,
+                                                     value: String)
```

Comment:
The parameter `value` is a `String` because in DHS, values are sent as strings. Does this make sense for GDS?